.github/workflows/cleanup.sh

```diff
@@ -2,6 +2,7 @@
 
 set -o nounset
 
+test -z ${FATS_DIR-} && FATS_DIR=`dirname "${BASH_SOURCE[0]}"`/../..
```

Comment:
rm

this script should never be invoked by anything other than the CI workflow.

Reply:
Why not? It's useful for testing. Seems harmless.

Reply:
this script is not part of FATS and may change at any time without notice. The harm is that the value is expected to be provided externally. By defaulting the value, we may not notice if the value isn't set, but should have been.

You can use these scripts as example for writing your own scripts that consume FATS, but these scripts should not be consumed directly. If you really want this behavior, copy the script and add it to your local copy.